### ISSUE TITLE
Not work in AWS Lambda

### ISSUE THREAD

**Issue by ange007** [Yoctol/bottender#648]
Hello.
I try setup bot in AWS Lambda, but bot _(Telegram)_ functions are not working.

Example:
```js
module.exports = async function App( context ) {
	const event = context.event;
	
	console.log(event);
	console.log(event.message.from);
	console.log(event.message.chat);

	await context.sendText( `@${event.message.from.username} - Hi!` );
```

Lambda log:
![image](https://user-images.githubusercontent.com/2736539/74108065-ec430880-4b7e-11ea-9abb-2995138d0cdf.png)

In local that works well.
What could be the reason?

**Comment by HcwXd** [comment]
Hi @ange007 

I have tested some cases and here are some of my findings. Let me know if you have any further question or information 😅 


### tldr:
The AWS Lambda functions seem to have problem sending requests to the webhook recently. Since the problem had never occurred before when I developed using Lambda + Bottender, I may wait for a few days and check if the problem is still occurring. 

### Test cases:
1. AWS + Telegram
2. AWS + LINE
3. AWS + Messenger
4. GCP + Telegram
5. GCP + LINE
6. GCP + Messenger

### Result:
AWS cases all fail to send text to user.
GCP cases all work as expected.

### Detailed failed cases:
The AWS Lambda functions now can't send request to the webhook correctly. 

Sometimes there's no error message shown and sometimes there're some types of error message will display randomly (See screenshot 1 & 2). 

However, when you trigger the function multiples times in a short period, the AWS Lambda functions will actually send the first few requests to the webhook successfully but fail to send the latter ones (See screenshot 3).

#### Screenshot 1

![image](https://user-images.githubusercontent.com/32091985/74187208-0cf68580-4c87-11ea-9ac8-d262a81005ad.png)

#### Screenshot 2

![image](https://user-images.githubusercontent.com/32091985/74187256-25ff3680-4c87-11ea-9154-82719f113a7f.png)


#### Screenshot 3

![image](https://user-images.githubusercontent.com/32091985/74187320-462ef580-4c87-11ea-90e8-edd3b371aebf.png)

**Comment by ange007** [comment]
Hi, @HcwXd 
Maybe need wrote to AWS support?
But i don`t know what write to them.

**Comment by ange007** [comment]
Hi.
Any idea?
Perhaps need add connection settings for the bot? With the ability to change parameters? For fix connection errors?

**Comment by ange007** [comment]
Has anyone come across this?
It doesn’t work for me.

**Comment by michaelwills** [comment]
Quick note @ange007. Have you tried setting `sync: true` for the bot's config in bottender.config? I am only testing locally at the moment but it will immediately return without that setting.

**Comment by kaito3desuyo** [comment]
I have same issue but message is diffrent.
![image](https://user-images.githubusercontent.com/14922052/107898207-8c93f000-6f7e-11eb-9ec0-6e7aa0f350b2.png)

**Comment by kaito3desuyo** [comment]
I have same issue but message is different.

![無題](https://user-images.githubusercontent.com/14922052/107898367-e399c500-6f7e-11eb-8ea1-c97cc1475a44.png)
